Week2/homework/exercise4/queries.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+let queries = [
+    {
+        query : 'SELECT d.title "Department", COUNT(e.full_name) "Number_of_Employees" FROM department d ' +
```

Comment:
I think it'd be more efficient to `COUNT(e.employee_no)` because it's the primary key.